bootstrap.py

```diff
@@ -714,6 +707,7 @@ class MEOptions:
     parser.add_option("-l", "--login", dest="login", default='admin', help="Login user for API Calls", metavar="LOGIN")
     parser.add_option("-p", "--password", dest="password", help="Password for specified user. Will prompt if omitted", metavar="PASSWORD")
     parser.add_option("--legacy-login", dest="legacy_login", default='admin', help="Login user for Satellite 5 API Calls", metavar="LOGIN")
+    parser.add_option("--fqdn", dest="fqdn", help="Set an explicit FQDN, overriding detected FQDN from socket.getfqdn()", metavar="OVERRIDE_FQDN", default=socket.getfqdn())
```

Comment:
how about adding `%default` to the help string, thus allowing the user to know what `socket.getfqdn()` detected beforehand? (see https://docs.python.org/2/library/optparse.html#generating-help)